test/test_package_descriptor.py

```diff
@@ -121,7 +121,7 @@ def test_magic_methods():
 
     d2.path = Path('/other/path')
     assert d1 != d2
-    assert hash(d1) != hash(d2)
+    # Not checking the hash.
```

Comment:
Either just remove the comment or describe why the hash isn't checked. I would think the first is sufficient - it is commonly not being documented what is *not* tested.